os/selinux.md

```diff
@@ -0,0 +1,39 @@
+# Using SELinux on CoreOS
+
+SELinux is a fine-grained access control mechanism that is integrated into
+CoreOS and rkt. Each container is run in its own independent SELinux
+context, increasing isolation between containers even in the event of a
+container being compromised. CoreOS implements SELinux, but currently does
+not enforce SELinux protections by default.
+
+## Determining whether containers are compatible with SELinux policy
+
+To verify whether the current SELinux policy would inhibit your containers,
+enable SELinux logging by running the following commands as root:
+* rm /etc/audit/rules.d/80-selinux.rules
+* rm /etc/audit/rules.d/99-default.rules
```

Comment:
Oh, also. those are symlinks and will get recreated on reboot iirc. replacing them with links to /dev/null or empty files would be needed to persistently remove them.